apps/condo/domains/news/schema/NewsItemScope.js

```diff
@@ -78,17 +80,30 @@ const NewsItemScope = new GQLListSchema('NewsItemScope', {
 
     },
     hooks: {
-        validateInput: async (args) => {
-            const { resolvedData, existingItem, context, operation } = args
-            const possibleItemData = { ...existingItem, ...resolvedData }
-
-            if (!get(possibleItemData, 'property') && !get(possibleItemData, 'unitType') && !get(possibleItemData, 'unitName')) {
-                throw new GQLError(ERRORS.EMPTY_NEWS_ITEM_SCOPE, context)
+        resolveInput: async ({ operation, existingItem, resolvedData }) => {
```

Comment:
it's better to move this resolve inside field

Reply:
or at least group this logic inside some function

Reply:
Moved to the field's hook